fava/templates/_tree_table.html

```diff
@@ -35,7 +35,8 @@
   {% set balance_children = account.balance_children|cost_or_value(ledger.end_date) %}
   {% set cost = account.balance|cost if g.conversion == 'at_value' else {} %}
   {% set cost_children = account.balance_children|cost if g.conversion == 'at_value' else {} %}
-  <li{{ ' class=toggled' if ledger.accounts[account.name].meta.get('fava-collapse-account') else '' }}>
+  {% set collapse = account.name|collapse_account_at_level(loop.depth0) %}
```

Comment:
Since this is used only once, maybe it's better to just use it in-place

Reply:
I'm not sure how would go about testing the feature in that case, so I have kept it as a filter.

Reply:
I meant to not declare a variable here and use the value directly below

Reply:
Ahh, right :smile: